### PR TITLE
docs: add notebook to create users

### PR DESCRIPTION
**Pull request by PierreLeveau**
Co-authored-by: Przemysław Czuba <Przemyslaw.czuba@kili-technology.com>

### CODE REVIEW

File: recipes/basic_project_setup.ipynb [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kili = Kili(api_key=KILI_API_KEY)\n",
+    "# Note: if you are using the on-premise version of Kili, you can specify the api endpoint here."
```
Comment (xczuba) [inline recipes/basic_project_setup.ipynb:123]: KILI_API_ENDPOINT = "https://cloud.kili-technology.com/api/label/v2/graphql"

File: recipes/basic_project_setup.ipynb [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+   "outputs": [],
+   "source": [
+    "project_id = result['id']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's add some assets to be labeled. Since this is just an example, let's use some free off-the-shelf examples from the Internet."
```
Comment (xczuba) [inline recipes/basic_project_setup.ipynb:201]: I've just realized I messed up this sentence. PLs change it to:

"We'll use some free off-the-shelf examples from the Internet"

File: recipes/basic_project_setup.ipynb [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we'll learn how to set up a basic Kili project.\n",
+    "\n",
+    "Here are the steps that we must follow:\n",
+    "\n",
+    "1. [Installing and instantiating Kili](#install-instantiate)\n",
```
Comment (xczuba) [inline recipes/basic_project_setup.ipynb:18]: Don't know why, but for some reason these links don't seem to work correctly on Git. I guess we'll have to hope they use these notebooks locally...

Comment (PierreLeveau) [reply recipes/basic_project_setup.ipynb]: I checked locally, it was not working in Vscode. I changed it to this and it worked:
```
1. [Installing and instantiating Kili](#-Installing-and-instantiating-Kili)
2. [Creating a basic Kili project](#-Creating-a-basic-Kili-project)
3. [Adding assets to project](#-Adding-assets-to-project)
4. [Adding users to project](#-Adding-users-to-project)
```

File: recipes/basic_project_setup.ipynb (hunk 2 above)
Comment (PierreLeveau) [reply recipes/basic_project_setup.ipynb]: done

**Pull request merged**